receiver/k8sclusterreceiver/watcher.go

```diff
@@ -98,8 +97,8 @@ func (rw *resourceWatcher) prepareSharedInformerFactory() {
 	rw.setupInformers(&appsv1.ReplicaSet{}, factory.Apps().V1().ReplicaSets().Informer())
 	rw.setupInformers(&appsv1.StatefulSet{}, factory.Apps().V1().StatefulSets().Informer())
 	rw.setupInformers(&batchv1.Job{}, factory.Batch().V1().Jobs().Informer())
-	rw.setupInformers(&batchv1beta1.CronJob{}, factory.Batch().V1beta1().CronJobs().Informer())
-	rw.setupInformers(&v2beta1.HorizontalPodAutoscaler{},
+	rw.setupInformers(&batchv1.CronJob{}, factory.Batch().V1beta1().CronJobs().Informer())
+	rw.setupInformers(&autoscalingv2beta2.HorizontalPodAutoscaler{},
 		factory.Autoscaling().V2beta1().HorizontalPodAutoscalers().Informer(),
```

Comment:
I believe that informer passed as the second argument supposed to match the k8s object type

Reply:
ack